librato/__init__.py

```diff
@@ -243,12 +243,28 @@ def set_tags(self, d):
     def add_tags(self, d):
         self.tags.update(d)
 
+    # Return all items for a "list" request
+    def _get_paginated_results(self, entity, klass, **query_props):
```

Comment:
Should we just make an `Iterable` class that all paginatable (word?) classes inherit from?  Then have the `list()` be return a generator?